controllers/controller.js

```diff
@@ -10,4 +11,19 @@ exports.getReviews=(req,res)=>{
     return selectReviews().then((review)=>{
         res.status(200).send({reviews:review})
     })
+}
+
+exports.getReviewByID=(req,res,next)=>{
+    const id=req.params.review_id
+    if(id%1!=0){
```

Comment:
Good thinking, but in the case of an invalid review_id - SQL will handle this for us and reject it. Have a look at the error code response and then how we can handle it.